native/src/components/SprungbrettListItem.tsx

```diff
@@ -18,7 +18,7 @@ class SprungbrettListItem extends PureComponent<SprungbrettListItemProps> {
     const { language, job, openJobInBrowser } = this.props
     return (
       <ListItem
-        thumbnail={<SprunbrettIcon width={25} height={25} />}
+        thumbnail={<SprungbrettIcon width={24} height={24} />}
```

Comment:
Sorry, might have been not that clear here:
Our custom icon component should be used which automatically sets the icon size to the default of 24px.
```suggestion
        thumbnail={<Icon Icon={SprungbrettIcon} />}
```